taxonomies/categories.txt

```diff
@@ -64805,6 +65310,7 @@ he:רטבים
 hr:umaci, umak
 hu:Szószok
 it:Salse
+ja:
```

Comment:
you forgot to paste the word, I guess

Reply:
Yes indeed. [Fixed](https://github.com/Naruyoko/openfoodfacts-server/commit/32e915cf5730fa0d4f78eda7238393861edd7d1b).